android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -579,9 +578,8 @@ public void setManifest(String manifestUrl, final RawManifest manifest, final St
       mReactRootView.loadVersion(mDetachSdkVersion).construct(ExperienceActivity.this);
       setReactRootView((View) mReactRootView.get());
 
-      String id;
       try {
-        id = Exponent.getInstance().encodeExperienceId(mExperienceIdString);
+        Exponent.encodeExperienceId(mExperienceKey.getLegacyId());
```

Comment:
Do we need this? Was the intent to make sure the scope key is URL-encodable? (For example, ScopedContext's constructor URL-encodes the scope key.)